pkg/cmd/api/api.go

```diff
@@ -365,13 +370,12 @@ func processResponse(resp *http.Response, opts *ApiOptions, bodyWriter, headersW
 
 	if opts.FilterOutput != "" && serverError == "" {
 		// TODO: reuse parsed query across pagination invocations
-		err = export.FilterJSON(bodyWriter, responseBody, opts.FilterOutput)
+		err = jq.Evaluate(responseBody, bodyWriter, opts.FilterOutput)
 		if err != nil {
 			return
 		}
 	} else if opts.Template != "" && serverError == "" {
-		// TODO: reuse parsed template across pagination invocations
```

Comment:
`Execute` no longer parses the template again so this is a small improvement to this code path.